<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_search"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_search">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_movie_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_movie_type">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>7</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>筛选</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="4">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>语言</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="4">
            <widget class="QCheckBox" name="checkBox_7">
             <property name="text">
              <string>挪威语</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>日语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QCheckBox" name="checkBox_16">
             <property name="text">
              <string>科萨语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_">
             <property name="text">
              <string>普通话</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>俄语</string>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QCheckBox" name="checkBox_18">
             <property name="text">
              <string>祖鲁语</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>方言</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>德语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>英语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QCheckBox" name="checkBox_17">
             <property name="text">
              <string>泰语</string>
             </property>
            </widget>
           </item>
           <item row="3" column="6">
            <widget class="QCheckBox" name="checkBox_15">
             <property name="text">
              <string>斯瓦希里语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QCheckBox" name="checkBox_19">
             <property name="text">
              <string>粤语</string>
             </property>
            </widget>
           </item>
           <item row="3" column="7">
            <widget class="QCheckBox" name="checkBox_20">
             <property name="text">
              <string>意大利语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QCheckBox" name="checkBox_21">
             <property name="text">
              <string>印地语</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QCheckBox" name="checkBox_22">
             <property name="text">
              <string>波兰语</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>地区</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkBox_25">
             <property name="text">
              <string>中国台湾</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_23">
             <property name="text">
              <string>中国大陆</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="checkBox_28">
             <property name="text">
              <string>日本</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QCheckBox" name="checkBox_29">
             <property name="text">
              <string>泰国</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QCheckBox" name="checkBox_27">
             <property name="text">
              <string>英国</string>
             </property>
            </widget>
           </item>
           <item row="5" column="5">
            <widget class="QCheckBox" name="checkBox_31">
             <property name="text">
              <string>意大利</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QCheckBox" name="checkBox_26">
             <property name="text">
              <string>美国</string>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QCheckBox" name="checkBox_30">
             <property name="text">
              <string>德国</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="checkBox_24">
             <property name="text">
              <string>中国香港</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QCheckBox" name="checkBox_32">
             <property name="text">
              <string>   挪威</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>电影类型</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkBox_37">
             <property name="text">
              <string>传记</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_34">
             <property name="text">
              <string>战争</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_36">
             <property name="text">
              <string>剧情</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_35">
             <property name="text">
              <string>历史</string>
             </property>
            </widget>
           </item>
           <item row="2" column="9">
            <widget class="QCheckBox" name="checkBox_46">
             <property name="text">
              <string>功夫</string>
             </property>
            </widget>
           </item>
           <item row="2" column="10">
            <widget class="QCheckBox" name="checkBox_48">
             <property name="text">
              <string> 灾难</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QCheckBox" name="checkBox_45">
             <property name="text">
              <string>艺术</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_33">
             <property name="text">
              <string>爱情</string>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QCheckBox" name="checkBox_47">
             <property name="text">
              <string> 古装</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="checkBox_38">
             <property name="text">
              <string>动画</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="checkBox_39">
             <property name="text">
              <string>科幻</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QCheckBox" name="checkBox_40">
             <property name="text">
              <string>冒险</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QCheckBox" name="checkBox_41">
             <property name="text">
              <string> 奇幻</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QCheckBox" name="checkBox_42">
             <property name="text">
              <string>悬疑</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QCheckBox" name="checkBox_44">
             <property name="text">
              <string>犯罪</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QCheckBox" name="checkBox_43">
             <property name="text">
              <string> 恐怖</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_chick">
         <property name="text">
          <string>按热度排名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_date">
         <property name="text">
          <string>按上映日期排名</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>8</horstretch>
        <verstretch>8</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit_commond_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit_movie_info">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_apend_commond">
        <property name="text">
         <string>追加评轮</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
